templates/agenda/show.html.twig

```diff
@@ -15,22 +15,22 @@
             {% elseif agenda.agendaItems|length > 0 %}
                 <button class="btn btn-danger" disabled data-toggle="tooltip" data-placement="auto" title="{% trans %}Cannot delete agenda containing agenda items{% endtrans %}">{% trans %}Delete agenda{% endtrans %}</button>
             {% else %}
-                <form id="myform" class="btn" method="post" action="{{ path('agenda_delete', {'id': agenda.id}) }}" onsubmit="return confirm('{% trans %}Are you sure you want to delete this agenda?{% endtrans %}');">
+                <form class="btn" method="post" action="{{ path('agenda_delete', {'id': agenda.id}) }}" onsubmit="return confirm('{% trans %}Are you sure you want to delete this agenda?{% endtrans %}');">
```

Comment:
This will break if the translation (or the translation source for that matter) contains `'`. You should use something like
```
… onsubmit="return confirm({{ 'Are you sure you want to delete this agenda?'|trans|json_encode|raw }})" …
```
instead to ensure proper escaping of special characters in the confirm message.

Linguistically it does not make sense to reply “Cancel” or ”OK” to a yes-no question, but we'll leave that for now. We should probably find a library to help us ask proper yes-no questions before triggering an action.

Reply:
Good catch. This is something that will be needed in many other places, so we will fix it in a different branch.